gsa/src/gmp/models/__tests__/filter.js

```diff
@@ -20,6 +20,7 @@ import {isArray} from '../../utils/identity';
 
 import Filter, {UNKNOWN_FILTER_ID} from '../filter';
 import FilterTerm from '../filter/filterterm';
+import {tsExternalModuleReference} from '@babel/types';
```

Comment:
It seems vscode added some additional import accidentially. Could you remove this one?

```suggestion
```